processor/README.md

```diff
@@ -269,34 +274,45 @@ the collector. Given that the amount and type of data a collector processes is
 environment specific and resource utilization of the collector is also dependent
 on the configured processors, it is important to put checks in place regarding
 memory usage. The memory_limiter processor offers the follow safeguards:
-- Ability to configure a ballast, which allocates memory and provides stability
-to the heap. In summary, the ballast increases the base size of the heap so that
-GC triggers are delayed and the number of GC cycles over time is reduced.
-- Ability to define a limit on memory usage at which time GC will be
-triggered to reduce memory consumption.
-- Ability to define a memory delta between two times which if exceed will trigger
-a GC to reduce memory consumption.
+- Ability to define an interval when memory usage will be checked and if memory
+usage exceeds a defined limit will trigger GC to reduce memory consumption.
+- Ability to define an interval when memory usage will be compared against the
+previous interval's value and if the delta exceeds a defined limit will trigger
+GC to reduce memory consumption.
+
+In addition, there is a command line option which can be used to define a ballast,
+which allocates memory and provides stability to the heap. If defined, the ballast
+increases the base size of the heap so that GC triggers are delayed and the number
+of GC cycles over time is reduced. While the ballast is configured via the command
+line, today the same value configured on the command line must also be defined in
+the memory_limiter processor.
 
 Note that while these configuration options can help mitigate out of memory
 situations, they are not a replacement for properly sizing the collector. Be
 aware that the ballast reserves memory that could otherwise be used for purposes
 including serving configured queues. In addition, if the limit or spike thresholds
-are crossed, the triggered GC will result in dropped data.
+are crossed, the triggered GC may result in dropped data.
```

Comment:
```suggestion
are crossed, the triggered GC may result in dropped data depending on the pipeline configuration.
```
My intent here is that in a proper configuration, the memory limiter will trigger the receiver to reply error to clients, and that should trigger a later retry so no actual data loss in that case.

Reply:
I added this a bit later for exactly that reason -- let me know if you think it should be repeated here.

The memory_limiter
processor should be the first processor defined in the pipeline (immediately after
the receivers). This is to ensure that backpressure can be sent to applicable
receivers and minimize the likelihood of dropped data when the memory_limiter kicks
in.